src/main/java/io/fusionauth/domain/search/EntitySearchCriteria.java

```diff
@@ -33,6 +33,9 @@ public void prepare() {
 
     orderBy = orderBy.replace("insertInstant", "insert_instant")
                      .replace("lastUpdateInstant", "last_update_instant");
+
+    // When sorting by 'name' in elasticsearch, we will want to use the raw/keyword mapping as opposed to the tokenized 'text' field
+    sortFields.stream().filter(sf -> sf.name.equals("name")).findFirst().ifPresent(sf -> sf.name = "name.raw");
```

Comment:
I think this belongs in the service layer instead of the public domain. Probably in `ElasticsearchEntitySearchEngine.prepareQuery`.

Reply:
Yes! Thankyou - I didn't feel comfortable putting it here.